src/components/Personalization/utils/addRenderToExecutedDecisions.js

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { assign } from "../../../utils";
+
+export default decisions => {
+  return decisions.map(decision => assign({ rendered: true }, decision));
+};
```

Comment:
This module should probably be named consistently with whatever the property ends up being named. Right now the property is named `rendered`, which means the module should probably be named `addRenderedToExecutedDecisions` rather than `addRenderToExecutedDecisions`.

Reply:
Agree, but let's decide on the name first!